management/src/main/java/org/kaazing/gateway/management/monitoring/configuration/MonitoringDataManagerInjector.java

```diff
@@ -21,18 +21,16 @@
 
 package org.kaazing.gateway.management.monitoring.configuration;
 
-import org.kaazing.gateway.management.monitoring.entity.factory.MonitoringEntityFactory;
-
 
 /**
- * This interface represents the factory for returning the specific MonitoringEntityFactory
+ * This interface represents the factory for returning monitoring data implementations
  * based on which types can be instantiated
  */
-public interface MonitoringEntityFactoryInjector {
+public interface MonitoringDataManagerInjector {
 
     /**
-     * Creates the monitoring entity factory
-     * @return the monitoring entity factory
+     * Creates the monitoring data manager
+     * @return
      */
-    MonitoringEntityFactory makeMonitoringEntityFactory();
+    MonitoringDataManager makeMonitoringDataManager();
```

Comment:
I can't find anywhere where this method is called. If it is dead code please remove this interface and the implementations.